### PR TITLE
Array $this->data

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

**Comment by mbrodala** [comment]
Why this change? The current text is actually just fine ...

**Comment by franzholz** [comment]
You have a big black double quote symbol. This is only for quotes.

The array $this->data provides the following data:

        The row of the currently edited record in $this->data['databaseRow']
        The configuration from the TCA in $this->data['parameterArray']['fieldConf']['config']
        The name of the input field in $this->data['parameterArray']['itemFormElName']
        The current value of the field in $this->data['parameterArray']['itemFormElValue']
        The JavaScript function to be called on changing the field in $this->data['parameterArray']['fieldChangeFunc']

In order ...

**Comment by mbrodala** [comment]
OK, so reducing the indentation would be enough and we can keep the current text.

**Review by franzholz — COMMENTED** [review]
missing part of ::

File: Documentation/ColumnsConfig/Type/User/Index.rst [hunk 1]
```diff
@@ -161,8 +161,10 @@ implementing a rendering. See :ref:`FormEngine docs
          <t3core:security-xss>`.
 
 
-   The array :php:`$this->data` provides the following data:
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User/Index.rst:164]: Please restore the original text ...

File: Documentation/ColumnsConfig/Type/User/Index.rst [hunk 2]
```diff
@@ -161,8 +161,10 @@ implementing a rendering. See :ref:`FormEngine docs
...
-
+   Array :php:`$this->data`: ...
+   
+   ::
+   
       * The row of the currently edited record in
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User/Index.rst:168]: ... and just reduce the indentation here.

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ColumnsConfig/Type/User/Index.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/ColumnsConfig/Type/User/Index.rst:164]: The original text gives no information. Why not make it shorter?

File: Documentation/ColumnsConfig/Type/User/Index.rst (hunk 2 above)
Comment (franzholz) [reply Documentation/ColumnsConfig/Type/User/Index.rst:168]: Shouldn't the block remain?

File: Documentation/ColumnsConfig/Type/User/Index.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/User/Index.rst:164]: The current version is better suited for humans. ;-)

File: Documentation/ColumnsConfig/Type/User/Index.rst (hunk 2 above)
Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/User/Index.rst:168]: Actually no, a plain list is enough. (Technically a table would be better but that's too much change at once.)

File: Documentation/ColumnsConfig/Type/User/Index.rst [hunk 3]
```diff
@@ -161,18 +161,18 @@ implementing a rendering. See :ref:`FormEngine docs
...
-        :php:`$this->data['databaseRow']`
-      * The configuration from the TCA in
-        :php:`$this->data['parameterArray']['fieldConf']['config']`
-      * The name of the input field in
-        :php:`$this->data['parameterArray']['itemFormElName']`
-      * The current value of the field in
-        :php:`$this->data['parameterArray']['itemFormElValue']`
-      * The JavaScript function to be called on changing the field in
-        :php:`$this->data['parameterArray']['fieldChangeFunc']`
+   The array :php:`$this->data` consists the following parts:
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User/Index.rst:164]:
```suggestion
   The array :php:`$this->data` consists of the following parts:
```

**Review by mbrodala — APPROVED** [review]

**Pull request merged**